package.json

```diff
@@ -21,19 +21,18 @@
   ],
   "license": "MIT",
   "dependencies": {
-    "go-ipfs": "0.3.6",
-    "ipfs-api": "2.2.1",
-    "kew": "0.5.0",
-    "lodash": "3.6.0",
+    "go-ipfs": "0.3.7-3",
```

Comment:
I'm not sure but I'll fix it if it works. I'll I've to check the semver calculator when I get home. 

--
Bret Comnes
http://bret.io
Sent from my mobile

> On Sep 25, 2015, at 1:05 PM, Juan Benet <notifications@github.com> wrote:
> 
> In package.json:
> 
> > @@ -21,19 +21,18 @@
> >    ],
> >    "license": "MIT",
> >    "dependencies": {
> > -    "go-ipfs": "0.3.6",
> > -    "ipfs-api": "2.2.1",
> > -    "kew": "0.5.0",
> > -    "lodash": "3.6.0",
> > +    "go-ipfs": "0.3.7-3",
> do we need to be specific? can we not say 0.3.7 and get the (only) one that's published? i checked this manually with npm install 0.3.7 and it installed one... but not with a dep)
> 
> —
> Reply to this email directly or view it on GitHub.
>